examples/how-to-simple-python.md

```diff
@@ -80,7 +81,7 @@ def get_process_by_id(pid):
           sys.exit()
   return resp.json()
```

Comment:
Should line 81 and 82 be part of the hidden code from 66-80

Reply:
@hal9000-hub sorry, I'm not sure I get your comment. Can you please elaborate?

Reply:
Line 81 (sys.exit) and 82 (return resp.json) should be part of the function "get_process_by_id", so those two lines should also be hidden in (@@ -80,7 +81,7 @@ def get_process_by_id(pid):)

Reply:
@hal9000-hub they are already part of the function `get_process_by_id`. Am I missing something?